aiosip/contact.py

```diff
@@ -8,7 +8,7 @@
 
 
 # Regex pattern from p2p-sip project
-CONTACT_PATTERNS = [re.compile('^(?P<name>[a-zA-Z0-9\-\._\+~ \t]*)<(?P<uri>[^>]+)>(?:;(?P<params>[^\?]*))?'),
+CONTACT_PATTERNS = [re.compile('^(?P<name>["a-zA-Z0-9\-\._\+~ \t]*)<(?P<uri>[^>]+)>(?:;(?P<params>[^\?]*))?'),
```

Comment:
Yeah, I should have revisited this change before posting it. Its one of the first things I changed trying to get my registration logic working because it seemed like it wasn't handling quoted usernames correctly.

But it is, my stuff works fine just with this change dropped. I see correct username in generated traffic. Looking like this is the result of me not understanding how aiosip was working.